.github/workflows/build_rpms.yml

```diff
@@ -53,5 +73,7 @@ jobs:
       - name: Test RPM installation
         run: |
           pushd rpms
+          pushd noarch
```

Comment:
`pushd rpms/noarch`? Or is there a reason to split these into 2?

Reply:
Will fix